novawallet/Modules/WalletsList/Common/WalletsListInteractor.swift

```diff
@@ -38,16 +38,16 @@ class WalletsListInteractor {
     func applyWallets(changes: [DataProviderChange<ManagedMetaAccountModel>]) {
         basePresenter?.didReceiveWalletsChanges(changes)
     }
-}
 
-extension WalletsListInteractor: WalletsListInteractorInputProtocol {
     func setup() {
         subscribeChains()
         subscribeWallets()
         setupBalancesStore()
     }
 }
 
+extension WalletsListInteractor: WalletsListInteractorInputProtocol {}
```

Comment:
It might be better to cut off the extension completely and move protocol confirmation up to the class declaration